mappings/net/minecraft/entity/passive/HorseBaseEntity.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/class_1496 net/minecraft/entity/passive/HorseBaseEntity
 	FIELD field_18118 PARENT_HORSE_PREDICATE Lnet/minecraft/class_4051;
 	FIELD field_6955 temper I
-	FIELD field_6956 IS_BRED_HORSE Ljava/util/function/Predicate;
+	FIELD field_6956 BRED_HORSE Ljava/util/function/Predicate;
```

Comment:
Hmm, what does this predicate do?

Reply:
Used to set the parent predicate, so that child stay around their parent